client/app/scripts/superdesk/filters.js

```diff
@@ -106,5 +106,15 @@ define([
                     return 'danger';
                 }
             };
-        }]);
+        }])
+        .filter('mergeTargets', function() {
+            return function(array) {
+                var merged = [];
+                _.forEach(array, function(item) {
+                    merged.push(item.allow === false ? item.name : 'Not ' + item.name);
```

Comment:
Just checking - in the metadata widget template there is a snippet `t['allow'] == false ? 'Not ' + t.name : t.name`, which does the opposite from this line, i.e. prepending `'Not '` if the item is _not_ allowed. Is this the way it should be?

Also, `item.allow === false` could perhaps be written simply as `!item.allow`, but nothing significant.